src/handlers/localstorage/mmkvStorageBackend.ts

```diff
@@ -0,0 +1,29 @@
+import { MMKV } from 'react-native-mmkv';
+import { STORAGE_IDS } from '../../model/mmkv';
+
+const mmkvLocalStorage = new MMKV({
+  id: STORAGE_IDS.LOCAL_STORAGE_ADAPTER,
+});
+
+export const mmkvStorageBackend = {
+  clear() {
+    return new Promise(res => {
+      res(mmkvLocalStorage.clearAll());
+    });
+  },
```

Comment:
```suggestion
  clear() {
    mmkvLocalStorage.clearAll())
  },
```
This should be enough. I read the code of this lib and looks like it `supports` both async and sync mode so there's no need to "pretend" to have async logic.

Similar fix can be applied to the following functions